README.md

```diff
@@ -22,16 +42,26 @@ ts_m2fpga
 │   ├── ilcCommmunication.md
 ├── fpgaInterface
 │   ├── NiFpga_mainFPGA.h
-│   ├── NiFpga_mainFPGA.lvbitx
 │   ├── NiFpga.c
 │   ├── NiFpga.h
+│   ├── NiFpga_mainFPGA.lvbitx
+│   ├── bitfile
+│       ├── m2fpga_FPGATarget_mainFPGA_<random_characters>.lvbitx
```

Comment:
Why do you have two `lvbitx` here?

Reply:
NiFpga_mainFPGA.lvbitx is a version of the original bitfile generated after FPGA compilation. This copy will be added to version control. The fpgaInterface/bitfile directory will be added to the gitignore file.

Reply:
**NiFpga_mainFPGA.lvbitx** is a version of the original bitfile created after FPGA compilation. So, the **fpgaInterface/bitfile** directory will be added to the **.gitignore** file because is not necessary to keep it in version control.